backend/data/src/main/kotlin/io/tolgee/repository/notification/NotificationSettingRepository.kt

```diff
@@ -0,0 +1,20 @@
+package io.tolgee.repository.notification
+
+import io.tolgee.model.notifications.NotificationChannel
+import io.tolgee.model.notifications.NotificationSetting
+import io.tolgee.model.notifications.NotificationTypeGroup
+import org.springframework.context.annotation.Lazy
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+
+@Repository
+@Lazy
+interface NotificationSettingRepository : JpaRepository<NotificationSetting, Long> {
+  fun findByUserId(userId: Long): List<NotificationSetting>
+
+  fun findByUserIdAndGroupAndChannel(
```

Comment:
Head up. The user can be deleted or disabled. Write custom query everywhere you're working with user and filter our disabled and deteled users.

Reply:
Will be done in another PR.